viewing_party/party.py

```diff
@@ -1,23 +1,139 @@
 # ------------- WAVE 1 --------------------
 
+from tests.test_constants import USER_DATA_5
+import copy
+
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+    movie = {"title" : title,
+                "genre" : genre,
+                "rating" : rating}
+    return movie

```

Comment:
If you're going to return a value right away you don't need to assign it to a variable:

```suggestion
    return {"title" : title,
                "genre" : genre,
                "rating" : rating}
```